tools/migrate/main.go

```diff
@@ -254,22 +255,21 @@ func migrate(srcDir, dstDir string) error {
 			return errors.Trace(err)
 		}
 
-		index.Components[comp.Name] = manifest.ComponentItem{
-			Name:        comp.Name,
-			Description: comp.Desc,
```

Comment:
The description is used at `tiup list`, we must fetch all component manifest to display the description if we remove it from `ComponentItem`.

Reply:
I think it is ok - we will have all the manifests locally and we can cache the descriptions locally too.